app/views/CreateChannelView.tsx

```diff
@@ -94,11 +96,13 @@ interface ICreateChannelViewProps extends IBaseScreen<ChatsStackParamList, 'Crea
 	teamId: string;
 	createPublicChannelPermission: string[] | undefined;
 	createPrivateChannelPermission: string[] | undefined;
+	useRealName: boolean;
 }
 
 interface ISwitch extends SwitchProps {
 	id: string;
 	label: string;
+	hint: string;
 }
 
 class CreateChannelView extends React.Component<ICreateChannelViewProps, ICreateChannelViewState> {
```

Comment:
It's a redesign. It's expected to migrate all these screens of the flow to Hooks.